gapic-generator/templates/default/shared/_client.erb

```diff
@@ -97,13 +114,15 @@ class <%= service.name %>Client
     )
     @<%= service.name.underscore %>_stub = create_stub credentials, scopes
 
+<% unless api.configuration[:skip_credentials] %>
     defaults = default_settings client_config, timeout, metadata, lib_name, lib_version
+<% end %>
 
 <% service.methods.each do |method| %>
 <% method_name = method.descriptor.name.underscore %>
 @<%= method_name %> = Google::Gax.create_api_call(
 @<%= service.name.underscore %>_stub.method(:<%= method_name %>),
-defaults["<%= method_name %>"],
+CallSettings.new,
```

Comment:
Does this line need to be special cased with the `if service.name == "Echo"`? Or, maybe we should just always add that `CallSettings` trick until we do this part for real?

Reply:
@jbolinger Agreed, we got there by accident, I'll remove that conditional.